.travis.yml

```diff
@@ -6,8 +6,56 @@ cache:
   yarn: true
   directories:
   - node_modules
-  - packages/obojobo-document-engine/node_modules
-  - packages/obojobo-document-xml-parser/node_modules
-  - packages/obojobo-express/node_modules
 script:
 - yarn test:ci
+
+jobs:
+  include:
+    - stage: obojobo-express npm release
+      # only run this stage if its a tag that matches this a v1.23.3 pattern
+      if: tag =~ ^v[0-9]+\.[0-9]+\.[0-9]+
+      # run this on only one node platform (allows us to put multiple platforms above for testing)
+      node_js: '10.4.1'
+      # move into the desired package and back out for deploying
+      before_deploy:
+        - cd packages/obojobo-express
+      after_deploy:
+        - cd ../../
+       # deploy this package to npm
+      deploy:
+        provider: npm
+        email: iturgeon+npm@gmail.com
```

Comment:
Alright to use personal email address? Do we need to use a UCF address?

Reply:
this email is the email of the npm user that is deploying. There's plusses and minuses to going with an organization account vs an individual account.

This option is a few less hoops now for a few extra hoops later should I leave.   Multiple people have npm ownership over the repo, so any of the other owners can setup a shared secret with travis and use issue a PR to update this.

If you feel strongly about it I'd suggest splitting it off to an issue and going forward w/ this as is

Reply:
Made #753